clients/cloud/confluent-cli/confluent-cli-example.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-CONFIG_FILE=~/.ccloud/config
+CONFIG_FILE=~/.confluent/config
```

Comment:
I changed `.ccloud` -> `.confluent` everywhere in the `.clients/cloud` subfolder, but can change these ones back if necessary